src/com/esotericsoftware/yamlbeans/YamlReader.java

```diff
@@ -49,7 +49,7 @@
 public class YamlReader implements AutoCloseable {
 	private final YamlConfig config;
 	Parser parser;
-	private final Map<String, Object> anchors = new HashMap();
+	private final Map<String, Object> anchorMap = new HashMap();

```

Comment:
There's no reason to put the type in the name. I don't think this needs to be renamed.

Reply:
sure, I will remove this.

Reply:
I have removed this change in my recent commit (9af34d4582a71e8d8396fe805198a1952354f073)